src/Opdex.Platform.Infrastructure/Data/Handlers/Transactions/SelectTransactionForVaultProposalPledgeRewindQueryHandler.cs

```diff
@@ -0,0 +1,101 @@
+using AutoMapper;
+using MediatR;
+using Opdex.Platform.Common.Models;
+using Opdex.Platform.Domain.Models.TransactionLogs;
+using Opdex.Platform.Domain.Models.Transactions;
+using Opdex.Platform.Infrastructure.Abstractions.Data;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Extensions;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Models.Transactions;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Models.Transactions.TransactionLogs;
+using Opdex.Platform.Infrastructure.Abstractions.Data.Queries.Transactions;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Opdex.Platform.Infrastructure.Data.Handlers.Transactions;
+
+public class SelectTransactionForVaultProposalPledgeRewindQueryHandler
+    : IRequestHandler<SelectTransactionForVaultProposalPledgeRewindQuery, Transaction>
+{
+    private const string Split = "Split";
+
+    private static readonly string SqlQuery =
+        @$"SELECT
+                t.{nameof(TransactionEntity.Id)},
+                t.`{nameof(TransactionEntity.To)}`,
```

Comment:
nitpick: Occasional ` character in all of these queries

Reply:
the ` has to stay there for reserved SQL words.